src/api.v2/helpers/worker/workSubmit/submitEmbeddingWork.js

```diff
@@ -1,17 +1,17 @@
 const submitWork = require('./submitWork');
 
-
 const submitEmbeddingWork = async (message) => {
   const {
     experimentId, input:
-    { authJWT, config: { embeddingSettings: { method, methodSettings } } },
+    { authJWT, config: { embeddingSettings: { method, methodSettings, useSaved } } },
```

Comment:
When is this useSaved needed/used?

Reply:
I think it's something added when setting up seurat upload, ideally we would look into removing it but for now I just added this to match what is being sent from the UI. With this addition the embedding works again

Reply:
What does `useSaved` do?

Reply:
don't remember, all I know is in the context I provided

Reply:
Oh now i remember something, i think it was added by Alex, and its used only in the Seurat upload. Maybe you can ask him if we need this?

Reply:
@alexvpickering did you add `useSaved`? What do we use it for?

Reply:
Yes I added it for Seurat object upload -- it tells the worker not to re-calculate an embedding but rather to used the embedding that is saved in the Seurat object.